pages.es/common/dotnet-publish.md

```diff
@@ -0,0 +1,28 @@
+# dotnet publish
+
+> Publica una aplicación .NET y sus dependencias en una carpeta para despliegue de un sistema local.
+> Más información: https://docs.microsoft.com/dotnet/core/tools/dotnet-publish.
+
+- Compila un proyecto .NET en modo de lanzamiento:
+
+`dotnet publish --configuration Release {{ruta/al/archivo_del_proyecto}}`
```

Comment:
The argument options should be between `{{}}`:
```suggestion
`dotnet publish --configuration {{Release}} {{ruta/al/archivo_del_proyecto}}`
```
The same should be applied to the English page.